interfaces/xenmgr_vm.xml

```diff
@@ -225,6 +225,51 @@
     <property name="display-handler-strict" type="b" access="readwrite">
       <tp:docstring>Strict checking for information about a VM's domain specific information (color, name, nickname)</tp:docstring>
     </property>
+    <property name="image_path_vg" type="s" access="readwrite">
+      <tp:docstring>Qt Proxy class version</tp:docstring>
```

Comment:
These should have a more informative description

Reply:
It looks like most of these are not wired to anything (except `image_path_vg`). From the looks of it this is to offer some persistent UI configuration?
I can just remove them or document them as Unused if they have a future use.